src/payment/checkout/Checkout.jsx

```diff
@@ -64,6 +64,24 @@ class Checkout extends React.Component {
     );
   };
 
+  handleSubmitStripe = (formData) => {
+    console.log('[Zebra] handleSubmitStripe called');
```

Comment:
Do we have a message logger we can use instead of console.log and make these statements debug messages? If theyre all just temp message for the sandbox then ignore this statement.

Reply:
These are just temporary for the sandbox work